packages/internal/src/paths.ts

```diff
@@ -112,6 +111,8 @@ export const resolveFile = (
  */
 export const getPaths = (BASE_DIR: string = getBaseDir()): Paths => {
   const routes = resolveFile(path.join(BASE_DIR, PATH_WEB_ROUTES)) as string
+  const { schemaPath } = getConfig(getConfigPath()).api
```

Comment:
Could we make `getConfig()` also supply a default value for `schemaPath`

Reply:
The reason for wanting this is so that people don't have to supply a value in `redwood.toml` for it to still work normally.

Reply:
This one is defined within DEFAULT_CONFIG in `packages/internal/src/config.ts`

```suggestion
  schemaPath: './api/db/schema.prisma',
```

Reply:
Ah! I didn't realise we already had that!

Reply:
All the changes should be in with the latest commits below!